README.md

```diff
@@ -276,6 +276,15 @@ The above will not run StacktraceModelTest, as that is also matched by "test.exc
 
 Note that if UI-tests are supposed to be part of the filtered run the "uitests" profile needs to be specified as well. Otherwise the UI won't start up and so the tests fail.
 
+
+## Building using docker and docker-compose
+
+```
+docker-compose run -f docker/docker-compose.yml jmc
+```
+
+Once build has finished the results will be in the `target` directory
```

Comment:
For me, it's this command that works- `docker-compose -f docker/docker-compose.yml run jmc`

Reply:
That's effectively identical isn't it - just changing the order between `run` and `-f docker/docker-compose.yml` ?

Reply:
You were definitely correct that I'd missed `jmc` at the end previously

Reply:
I guess it should be the same but having run before the file does not work for me, maybe it's just the version of docker-compose I have (docker-compose version 1.23.2, build 1110ad01)

Reply:
Fair enough, I have no objection to changing that!

Reply:
I also might be wrong that it ever worked for me and just wrote what I thought I'd run.

It's fixed now though